website/docs/reference/resource-properties/constraints.md

```diff
@@ -11,6 +11,8 @@ Constraints require the declaration and enforcement of a model [contract](/refer
 
 **Constraints are never applied on `ephemeral` models or those materialized as `view`**. Only `table` and `incremental` models support applying and enforcing constraints.
 
+<VersionBlock lastVersion="1.8">
+
 ## Defining constraints
```

Comment:
the 'defining constraints' section is generally the same for 1.8 and 1.9 - would it make sense to instead version wrap the code snippet and 'When using foreign_key, you need to specify the referenced table's schema manually...' content for 1.8 and lower? Rather than version wrapping defining constraints twice (for 1.8 and lower and 1.9+)?